### ISSUE TITLE
Cannot get items from the open grave

### ISSUE THREAD

**Issue by tnevolin** [phobos2077/sfall#412]
[SLOT01.zip](https://github.com/phobos2077/sfall/files/7479245/SLOT01.zip)

# Test 1

Installed GOG version fresh.
Took shovel from Arroyo.
Traveled to Den.
Saved game (attached)
Dig up the grave on graveyard.
Interact with grave (hand cursor).
Inventory screen pops up so I can take items from the grave.

# Test 2

Installed sFall 4.3.1 over it (copy all dll and ini as instructed).
Loaded save.
Dig up the grave on graveyard.
Interact with grave (hand cursor).
Inventory screen *does not* pop up.

# Test 3 (indirect)

Previously I had game with FO2 + PRU + EcCo + FO2Tweaks installed.
They have updated sFall to version 3.4 as listed in ddraw.ini.
I didn't see this problem there.

# Test 4

Installed sfall_4.3.0.2 on top.
Bug disappeared!
Installed sfall_4.3.1 on top again.
Bug appeared again.

**Comment by NovaRain** [comment]
Sorry about the trouble. You can use the test build: [sfall-build_d4f0119.zip](https://github.com/phobos2077/sfall/files/7490658/sfall-build_d4f0119.zip) 
Or extract the temp fix script from [gl_gravetmpfix.zip](https://github.com/phobos2077/sfall/files/7490667/gl_gravetmpfix.zip) to your script folder and set AllowUnsafeScripting=1 in ddraw.ini, it will disable the fix added in 4.3.1 that caused the issue.

**Comment by tnevolin** [comment]
I have replaced DLL from sfall-build_d4f0119.zip and it fixed it. Thank you! Please keep this fix in future releases.

**Comment by FakelsHub** [comment]
I think we need to make a fix release.

**Comment by NovaRain** [comment]
The changelog will be pretty short.
```
>Changed the fix for grave type containers in 4.3.1 to an option, to fix compatibility with existing grave scripts
>(maybe mention the optimization for Glovz's and YAAM)
```
